### DISCUSSION TITLE
Anything I can help with?

### DISCUSSION THREAD

**Discussion by neilswann80** [eduardofilo/RG350_adam_image#202] [General]
@eduardofilo I'm keeping an eye on discussions and issues to see if there's anything I can help you with.  I was wondering if there's anything interesting you're working on with the image currently and if there's anything you'd like a hand with?

I've found a larger collection of PICO-8 games on archive.org, so when I have time I'll test those and see if I can get the `alias_pico-8.txt` expanded.  I created the last alias file manually which was dull work, but I figure I could create a script to generate one automatically from the `gameslist.xml` file Skraper creates on completion.  Perhaps also a script to merge alias files together avoiding duplicates.

**Comment by eduardofilo** [comment]
Oh thank you. As for the future of the image my plans are:

1. Keep an eye out for new developments in ODb, RA and SM to incorporate when there are major improvements.
2. Build a parameterizable system to assemble the image from all its sources, and not based on an instance of the previous image which is what the current script that builds it is based on.

The latter I wish I had started doing some time ago, but I had a family problem in October that has consumed my free time until mid-January (which is why 1.4 was delayed). This project also has the hidden objective that everyone can build their own image and I can dissociate myself a little bit from all this. Something similar to the RA installer, which since a few months ago is parameterizable and it is easy to build a custom one.
I have not yet been able to design the machinery to mount the image, so I still do not know how it will be possible to collaborate. Once it is assembled, it will be possible to contribute to the parameterization files of the sources.

Another topic I have pending is to study how to install the image through cable following this system:
https://boards.dingoonity.org/retrominirs-90/a-quick-guide-to-flashing-opendingux-on-the-rs-90-using-windows-10-x64/msg186557/

The previous project (point 2 in the former list) is a priority for me and will take time, so if you want to analyze this other one maybe it would help for the next image updates. Especially for minor updates like 1.4.1.

Thanks...
